.github/workflows/daily-e2e-tests-libvirt.yaml

```diff
@@ -0,0 +1,144 @@
+# (C) Copyright Confidential Containers Contributors 2023.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Daily run the e2e tests for libvirt.
+---
+name: daily e2e tests for libvirt
+
+on:
+  schedule:
+  # Runs "at 04:15(UTC time) every day" (see https://crontab.guru)
+  # will base on default branch `main`
+    - cron: '15 4 * * *'
```

Comment:
Not sure if github supports `H` but with cronjobs it's usually adviced to use `H` rather than picking up random minute values so that cron can distribute the workflows throughout the hour (`H 4 * * *`)

Reply:
I learned something new today :)

But I don't know whether github supports `H` or not. It claims to use the POSIX cron syntax (https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule) but not sure if all expressions.